Python/Module1_GettingStartedWithPython/GettingStartedWithPython.md

```diff
@@ -144,6 +144,39 @@ As such, Python is a language that is conducive to rapidly prototyping and testi
 We will be relying heavily on Python and a library for doing optimized numerical computations, called NumPy, throughout this course.
 <!-- #endregion -->
 
+<!-- #region -->
+## Keeping Up To Date
+
+New versions of Python come out periodically, bringing new features and fixes. 
+The most recent version of Python is Python 3.8.4.
+Unfortunately, it takes time for packages to support new versions of Python.
+As a result, many packages don't support Python 3.8, and will require you to install Python 3.7 or, in some cases, 3.6.
+
+All Python versions numbers use the A.B.C format. 
+The three numbers denote major releases, minor releases, and patches.
+
+- The first number denotes major releases to the language. 
+The most current major release is Python 3, but Python 2 is still in use. 
+When a major release comes out, it means that older code will not necessarily work with the new release. 
+Running Python 2 code in a Python 3 interpreter can yield errors, as can running Python 3 code in a Python 2 interpreter. 
+Python 2 is no longer supported, but the transition to Python 3 has been slow.
+When reading code from the Internet, always check to make sure it uses Python 3.
```

Comment:
I'd rearrange the order here

```suggestion
- The first number denotes major releases to the language. 
When a major release comes out, it means that older code will not necessarily work with the new release. 
The most current major release is Python 3. Running Python 2 code in a Python 3 interpreter can yield errors, as can running Python 3 code in a Python 2 interpreter. 
Python 2 is no longer supported, but the transition to Python 3 has been slow, so Python 2 is still in use in some places.
When reading code from the Internet, always check to make sure it uses Python 3.
```

I'm not sure we want to talk much about 2/3 since 2 is officially dead... @rsokl?

Reply:
It might be okay to talk about Python 2 here since it is the one handy example of switching between major versions. That being said, the convo about it should be brief.